datasets/dataset_factory.py

```diff
@@ -46,9 +47,11 @@ def create(dataset_config: Config.Dataset) -> typing.Tuple[DataLoader, DataLoade
         valid_length = int(dataset_config.validation_split * len(train_dataset))
         train_dataset, valid_dataset = random_split(train_dataset, [len(train_dataset) - valid_length, valid_length])
 
+        train_sampler, valid_sampler, test_sampler = None, None, None
         if dataset_config.oversampling == 'oversampling':
-            # TODO: Implement oversampling of the minority class
-            raise NotImplementedError('Oversampling of the minority class not implemented yet.')
+            train_sampler = get_balanced_sampler(train_dataset.dataset.labels[train_dataset.indices])
+            valid_sampler = get_balanced_sampler(valid_dataset.dataset.labels[valid_dataset.indices])
+            test_sampler = get_balanced_sampler(test_dataset.labels)
```

Comment:
The goal here is to re-sample an imbalanced dataset! One prob is that my imbalanced dataset do not have a labels attribute.